modules/git/repo_language_stats_gogit.go

```diff
@@ -55,7 +58,9 @@ func (repo *Repository) GetLanguageStats(commitID string) (map[string]int64, err
 
 		if checker != nil {
 			attrs, err := checker.CheckPath(f.Name)
-			if err == nil {
+			if err != nil {
+					log.Error("CheckPath returns error: %v", err)
```

Comment:
Why not
```go
return err
```
here?
So either
```suggestion
					log.Error("CheckPath returns error: %w", err)
					return err
```
or
```suggestion
					return err
```